### ISSUE TITLE
[a11y] Cannot save CO form with all required fields met

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1480]
![image](https://user-images.githubusercontent.com/37256050/167652155-13cef3fd-f0b7-4885-a733-2142d5d32726.png)

In the `fwri` database, I cannot save a CO form without adding an element to it even though all required fields are met.

`edge` is able to save without any data entered as all required fields (cataloger) are filled.

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/167885371-7e4733e5-a423-4bb9-9f49-2339d0b32052.png)

This record should be saveable without any change, right?

**Comment by maxxxxxdlp** [comment]
> This record should be saveable without any change, right?

Fixed
